tests/src/ExampleAnnotatedCommand.php

```diff
@@ -44,7 +44,7 @@ public function myEcho($one, $two = '', $multiple = [], $flip = false)
      * @usage bet alpha --flip
      *   Concatenate "alpha" and "bet".
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function myEchoAnnotated(InputInterface $input, OutputInterface $output)
```

Comment:
This method must be named `execute` for this test.

Reply:
OK, methods named execute will have to add a return type hint.

Reply:
Did that, seems green now, I guess old versions are passing because it is okay to add a function return type in an overriding class even if the base class didn't have it.